sdk/loadtestservice/load-testing-rest/samples-dev/sample.ts

```diff
@@ -41,34 +41,50 @@ async function main() {
     },
   });
 
+  if (testCreationResult.status !== "200" && testCreationResult.status !== "201") {
+    throw testCreationResult.body.error;
+  }
+
   // Uploading .jmx file to a test
-  await client.path("/loadtests/{testId}/files/{fileId}", testId, fileId).put({
-    contentType: "multipart/form-data",
-    body: {
-      file: readStream,
-    },
-  });
+  const fileUploadResult: any = await client
+    .path("/loadtests/{testId}/files/{fileId}", testId, fileId)
```

Comment:
will the call to create a test return the testId? If so I think we should use the result of the previous call instead of a random UUID